src/unstract/sdk/adapters/x2text/no_op/src/no_op_x2text.py

```diff
@@ -49,8 +52,7 @@ def process(
         )
         time.sleep(self.config.get("wait_time"))
         if output_file_path:
-            with open(output_file_path, "w", encoding="utf-8") as f:
-                f.write(extracted_text)
+            fs.write(extracted_text, encoding="utf-8")
```

Comment:
For better readability, we could resort to either always passing in arg_name or not.
Mixed approach would be confusing in my opinion.
Eg. fs.write(data=extracted_data, encoding="utf-8")
or
fs.write(extracted_data, "utf-8")

Reply:
I always prefer explicitly adding the kwarg name since we might not be clear enough with the variable names. Let's enforce kwargs as a standard

Reply:
Addressed